### DISCUSSION TITLE
Zarr-Python Core Devs - Please confirm your status

### DISCUSSION THREAD

**Discussion by jhamman** [zarr-developers/zarr-python#1608] [General]
The Zarr-Python core developer roster currently includes the following individuals:

### staying on

- @joshmoore (Josh Moore)
- @jni (Juan Nunez-Iglesias)
- @rabernat (Ryan Abernathey)
- @jhamman (Joe Hamman)
- @d-v-b (Davis Bennett)
- @jakirkham (jakirkham)
- @martindurant (Martin Durant)

### emeritus
- @alimanfoo (Alistair Miles)
- @shoyer (Stephan Hoyer)
- @ryan-williams (Ryan Williams)
- @jrbourbeau (James Bourbeau)
- @mzjp2 (Zain Patel)
- @grlee77 (Gregory Lee)

### stepping down
- @jeromekelleher (Jerome Kelleher)
- @tjcrone (Tim Crone)
- @funkey (Jan Funke)
- @shikharsg (shikharsg)
- @Carreau (Matthias Bussonnier)
- @dazzag24 (dazzag24)
- @WardF (Ward Fisher)

My impression is that most folks here are not able to participate in substantial ways with this project at this time. We are having trouble moving the project forward with so many inactive core developers. Two examples to this point include 1) we were unable to get some of the individuals above to share an email address for the core-developer google group, and 2) we are currently unable to approve the nomination of a new core developer. 

Therefore, I would like ask each of you above to confirm your status with the project as either: "active", "stepping down", or "emeritus". (I have indicated a few answers above for those that have already confirmed their status). 

*Note: I have proposed a change to the governance docs related to this issue: https://github.com/zarr-developers/governance/issues/38*

-----

_Update 2 Jan. 2024 - We did not hear from everyone so I have moved @alimanfoo to the "emeritus" category and @dazzag24 and @WardF to the stepping down category._

**Comment by shoyer** [comment]
Could you clarify the difference between "stepping down" and "emeritus"?

**Reply by jhamman** [reply]
The relevant section of the governance documents say:

> core developers can choose to become emeritus core developers and suspend  their approval and voting rights until they become active again.

So, if you would like to keep open the possibility of returning to the core dev team, you should opt for emeritus. 

Personally, I'd like you to stay closer to the project so please do opt for emeritus 🙏 .

**Reply by shoyer** [reply]
Sure, I'll take "emeritus" then :)

**Comment by jni** [comment]
Hi @jhamman,

I feel like I'm a bit in the middle — I am unable to participate often in day-to-day maintenance, and I'd be happy to step into an emeritus role if requested. However, I do try to respond when pinged and review major proposals on which I have meaningful things to say as they happen. So, anyway, do with that info what you will. 😅

**Reply by jhamman** [reply]
Hi @jni - I would personally be stoked if you stayed on. Are you able to access the core-dev google group? There is a new dev nomination pending there that could use your vote!

**Reply by jni** [reply]
Damn, sorry, I thought I said yes, but there was a send error between my Fastmail and my Google accounts. 🤦 If anyone has suggestions for non-Google mailing lists I would be super happy to hear them. 😅 I know about Framagroupes from French FOSS nonprofit Framasoft, but the standard hosting agreement is pretty onerous for an open project... 😅 https://framagroupes.org/abc/en/

**Reply by joshmoore** [reply]
I think I can request lists from the German research council if there are no other suggestions.

**Reply by jni** [reply]
To clarify, I wasn't necessarily advocating moving zarr-Python to a non-Google list (though I would be happy if we did that). I was just curious for myself and my other projects.

Fun factoid: you can't forward a normal email address to a googlegroups list — forwarded emails will always land in spam, and there is no way to whitelist them.

**Comment by martindurant** [comment]
I would like to stay on.
I wrote to @MSanKeys963 about being sure to be on the g-list, but I never saw the nomination email, so something got lost along the way.

**Reply by sanketverma1704** [reply]
Hi @martindurant.

I sent you an invite on Jun 21, 2023, but it says you didn't accept it, and the invite expired.

I've sent you the invite again. Please check your spam folder if it isn't in your inbox. 
Let me know if there are any issues. Thanks!

**Reply by martindurant** [reply]
(spam it was, d'oh)

**Reply by jhamman** [reply]
thanks @martindurant!

**Comment by tjcrone** [comment]
Stepping down. Thank you everyone!

**Comment by ryan-williams** [comment]
Emeritus sounds right for me, edited OP accordingly, thanks all.

**Reply by jhamman** [reply]
Thanks @ryan-williams! Don't go far, we want you back when time allows 😉

**Comment by funkey** [comment]
Stepping down. Thanks everyone!

**Reply by jhamman** [reply]
Thanks @funkey!

**Comment by shikharsg** [comment]
Stepping down. Thanks all!

**Reply by jhamman** [reply]
Thanks @shikharsg. Hope to see you around later on.

**Comment by joshmoore** [comment]
For completeness, staying on.

**Comment by mzjp2** [comment]
I'll take emeritus if there's no objection.

**Reply by jhamman** [reply]
Thanks @mzjp2!

**Comment by jakirkham** [comment]
Staying on

**Comment by Carreau** [comment]
I'm happy to step down, I can barely manages all the projects I'm on these days, and don't think I need any of the permissions.

Thanks for all you are doing !

note: Actually even better if I don't have permissions I almost clicked on "Close with comment" instead of just "Comment" because it's too early...

**Reply by jhamman** [reply]
Thanks @Carreau - don't go far!

**Comment by d-v-b** [comment]
staying on

**Comment by grlee77** [comment]
emeritus is good for me, thanks

**Comment by jhamman** [comment]
Documenting this discussion here: https://github.com/zarr-developers/zarr-python/pull/1628
